firmware/Sequencer.h

```diff
@@ -84,7 +89,8 @@ static void sequencer_init() {
 void sequencer_restart() {
   MIDI.sendRealTime(midi::Start);
   delay(1);
```

Comment:
not calling usbMIDI is a bug. They should both be called, which again proves that we need a higher level MIDI handler that manages the splitting/sending to multiple interfaces.

The 1 ms delay is to prevent overloading the MIDI interface. Being an ancient spec it's rated for 1000 messages per second so adding a delay of 1ms ensures we don't overload the bus. The higher level MIDI handler could take care of this but for now we need the delay. Could be reduced to 500 usec though.

Reply:
Added issue for the midi abstraction here: https://github.com/datomusic/duo-imxrt/issues/156
Maybe we can add the delay there too, but let's keep things like they are for now in this PR. I'll just add a line calling usbMIDI for the Start message also.